### PR TITLE
fix: Backup and restore integration test

### LINKED ISSUE #6223: Integration tests failed
* Link to run: https://github.com/keptn/keptn/actions/runs/1518985373
* Triggered by: Scheduled build
* Branch: master
* Version: 0.11.2-dev.202111290808
* Datetime: 202111290808
* Commit: 8cfa9fa7

Integration Tests have finished.

| Target Platform        |  Keptn Install    |  Keptn Auth  | Airgapped Images | Log Ingestion   | Log Forwarding   | Sequence States   | Sequence Timeout   | Sequence Control   | Sequence Queue   | Sequence Loop |  QG Standalone  |  Self Healing  |  Del-Assist  |  Backup-Restore  |  User-Man. Depl.  | Cont Deliv | Cont Deliv (Remote Exec) | Manage secrets | Webhook integration | Uniform Registration   |
| ---------------------- | ----------------- | ------------ | ---------------- | --------------- | ---------------- | ----------------- | ------------------ | ---------------- | ------------------ | ------------- | --------------- | -------------- | ------------ | ---------------- | ----------------- |----------- | ------------------------ | -------------- | ------------------- | ---------------------- |
| kubernetes/GKE 1.19 | success | success | skipped | success | success | success | success | success | success | success | success | success | success | :x: **failure** | success | skipped | success | success | success | success |
| kubernetes/GKE 1.21 | success | success | skipped | success | success | success | success | success | success | success | success | success | success | :x: **failure** | success | success | skipped | success | success | success |
| kubernetes/k3d-on-GHA v4.4.6 | success | success | success | skipped | skipped | skipped | skipped | skipped | skipped | skipped | skipped | skipped | skipped | skipped | skipped | skipped | skipped | skipped | skipped | skipped |
| kubernetes/k3s-on-GHA v1.16.15+k3s1 | success | success | skipped | success | success | success | success | success | success | success | success | success | skipped | skipped | skipped | skipped | skipped | success | success | success |
| kubernetes/k3s-on-GHA v1.21.1+k3s1 | success | success | skipped | success | success | success | success | success | success | success | success | success | skipped | skipped | skipped | skipped | skipped | success | success | success |
| openshift/minishift-on-GHA 3.11 | success | success | skipped | success | success | success | success | success | success | success | skipped | success | skipped | skipped | skipped | skipped | skipped | success | success | skipped |

Note: This issue was auto-generated from [integration_tests.yml](.github/workflows/integration_tests.yml)

### LINKED ISSUE #6239: Integration test backup_restore fails on timeout
`backuprestore_test.go` fails on timeout and instability. Every time the test is executes, it fails after restoring the data on the last step when verifying the deployment with `VerifyDirectDeployment()` function, no matter how long the timeout is or how many stages are used in the `shipyard.yaml`

### PR DESCRIPTION
**Pull request by odubajDT**
Signed-off-by: odubajDT <ondrej.dubaj@dynatrace.com>

## This PR

- fixed minor issue in backup_restore.go test regarding waiting for deployment in production for unstable version 0.1.1

### Related Issues

Fixes #6239

### CODE REVIEW

File: test/go-tests/backuprestore_test.go [hunk 1]
```diff
@@ -142,6 +126,13 @@ func Test_BackupRestore(t *testing.T) {
 	_, err = ExecuteCommandf("keptn delete project %s", keptnProjectName)
 	require.Nil(t, err)
 
+	//clientset, _ := keptnkubeutils.GetClientset(false)
+	//assert.Eventually(t, func() bool {
+	//	return wait.PollImmediate(time.Second*3, time.Minute*5, checkDeployment(clientset, serviceName, keptnProjectName+"-dev", WaitForDeploymentOptions{WithImageName: "ghcr.io/podtato-head/podtatoserver:v0.1.0"})) != nil
+	//}, time.Minute*time.Duration(10), time.Second)
+
+	//time.Sleep(10 * time.Minute)
```
Comment (thisthat) [inline test/go-tests/backuprestore_test.go:134]: please, remove this code if not needed

Comment (odubajDT) [reply test/go-tests/backuprestore_test.go:134]: sure, this is not prepared yet for review and merging

**Comment by odubajDT** [comment]
tests green: https://github.com/keptn/keptn/actions/runs/1524732430

**Comment by odubajDT** [comment]
Integration tests run for the current PR version are green: https://github.com/keptn/keptn/actions/runs/1524852896

**Review by bacherfl — APPROVED** [review]

**Pull request merged**